src/pages/MetricDetail.svelte

```diff
@@ -197,6 +225,15 @@
         {/each}
       </td>
     </tr>
+    {#if glamUrl}
```

Comment:
Argh, sorry, did some last minute testing and it looks like this doesn't do the right thing for events (it incorrectly suggests we can view them in GLAM). Maybe the suggestion below will fix that?

```suggestion
    {#if glamUrl && metric.type === "event"}
```

Reply:
Cool, thanks. I added a condition similar to one just above it. Let me know if that's correct. It worked locally for me.